dataikuapi/dss/ml.py

```diff
@@ -723,6 +723,44 @@ def get_partial_dependencies(self):
         return DSSPartialDependencies(data)
 
 
+class DSSSubpopulationGlobal(DSSExtensibleDict):
+    """
+    Object to read details of performance on global dataset used for subpopulation analyses.
```

Comment:
Trying to avoid "dataset", don't hesitate if you have a better wording

```suggestion
    Object to read details of performance on global population in a subpopulation analyses.
```